geode-core/src/main/java/org/apache/geode/internal/cache/GemFireCacheImpl.java

```diff
@@ -4246,7 +4231,7 @@ public void loadCacheXml(InputStream is)
 
       if (XML_PARAMETERIZATION_ENABLED) {
         char[] buffer = new char[1024];
-        reader = new BufferedReader(new InputStreamReader(is, "ISO-8859-1"));
+        reader = new BufferedReader(new InputStreamReader(is, StandardCharsets.ISO_8859_1));
```

Comment:
nice change.  Did the analyzer catch that?

Reply:
Yup!